include/dd/Package.hpp

```diff
@@ -1970,9 +2018,9 @@ namespace dd {
         }
 
         void printMatrix(const mEdge& e) {
-            unsigned long long element = 2u << e.p->v;
-            for (unsigned long long i = 0; i < element; i++) {
-                for (unsigned long long j = 0; j < element; j++) {
+            auto element = 2ULL << e.p->v;
```

Comment:
Can be `const`. Maybe make type explicit to avoid confusion of the resulting type of the left shift (`unsigned long long` in this case)?

Reply:
Can't be const. See above.